src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -15,6 +15,8 @@
 
     private GuiSettings guiSettings = new GuiSettings();
     private Path addressBookFilePath = Paths.get("data" , "addressbook.json");
+    private String blockSettings = "A,B,C,D";
+    private int roomSettings = 420;
```

Comment:
Perhaps `String[]` will be more suitable for `blockSettings` instead of `String`?
And `roomSettings` with `420` is quite confusing, can consider breakdown to min, max floor and room number.